meta-freescale-3rdparty-mbl/recipes-bsp/atf/atf-imx6ul-pico-mbl_git.bb

```diff
@@ -6,8 +6,7 @@ inherit mbl-artifact-names
 require recipes-bsp/atf/atf.inc
 
 # Replace from here:
-MBL_ATF_VERSION = "2.0"
-SRCREV_atf = "6a80c8bbc961ab55a562e0030247419e363286f7"
+MBL_ATF_VERSION = "2.1"
```

Comment:
You can drop this line because `MBL_ATF_VERSION` is being set in `meta-mbl-bsp-common/recipes-bsp/atf/atf.inc`.